packages/fast-components-react-base/src/horizontal-overflow/horizontal-overflow.tsx

```diff
@@ -118,7 +118,10 @@ class HorizontalOverflow extends Foundation<
             >
                 <div
                     style={{
-                        height: `${this.state.itemsHeight}px`,
+                        height:
+                            this.state.itemsHeight !== null
+                                ? `${this.state.itemsHeight}px`
+                                : "undefined",
```

Comment:
There is no such state as "undefined" for height, I think it should default to "auto" per https://developer.mozilla.org/en-US/docs/Web/CSS/height

Reply:
Sounds good, I'll update to auto now 👍